docker-compose.yaml

```diff
@@ -393,6 +393,28 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
+  full-node:
```

Comment:
Let's add a comment about what this full-node does: e.g. reads from the sequencer feed, forwards transactions to the sequencer.